libs/application/template-api-modules/src/lib/modules/templates/new-primary-school/new-primary-school.service.ts

```diff
@@ -14,19 +15,24 @@ export class NewPrimarySchoolService extends BaseTemplateApiService {
     super(ApplicationTypes.NEW_PRIMARY_SCHOOL)
   }
 
-  async getTypes({ auth }: TemplateApiModuleActionProps) {
-    return await this.friggClientService.getTypes(auth)
+  async getKeyOptionsTypes({ auth }: TemplateApiModuleActionProps) {
+    return await this.friggClientService.getKeyOptionsTypes(auth)
   }
 
   async getAllKeyOptions({ auth }: TemplateApiModuleActionProps) {
     return await this.friggClientService.getAllKeyOptions(auth, undefined)
   }
 
-  async getUserById({ auth }: TemplateApiModuleActionProps) {
-    return await this.friggClientService.getUserById(auth)
-  }
-
   async getAllSchoolsByMunicipality({ auth }: TemplateApiModuleActionProps) {
     return await this.friggClientService.getAllSchoolsByMunicipality(auth)
   }
+
+  async setChildInformation({
```

Comment:
Wouldn't it make more sense to be named as get?
```suggestion
  async getChildInformation({
```